django/contrib/admin/widgets.py

```diff
@@ -145,13 +145,13 @@ class ForeignKeyRawIdWidget(forms.TextInput):
     in a <select> box.
     """
     def __init__(self, rel, admin_site, attrs=None, using=None):
-        self.rel = rel
```

Comment:
I am personally using Widget.rel in some of my code. Some sort of compatibility would be nice if it's not too hard.